doc/install.rst

```diff
@@ -28,24 +28,96 @@ There are different ways to install scikit-learn:
+         ><span class="sk-expandable" data-packager="conda"><a href="https://docs.conda.io/projects/conda/en/latest/user-guide/install/">Install conda</a> (no administrator permission required).</span>
+       </div>
+
+Then run:
+
+.. raw:: html
+
+       <div class="highlight"><pre><code
+        ><span class="sk-expandable" data-packager="pip" data-os="linux" data-venv="">python3 -m venv sklearn-venv</span
+        ><span class="sk-expandable" data-packager="pip" data-os="windows" data-venv="">python -m venv sklearn-venv</span
+        ><span class="sk-expandable" data-packager="pip" data-os="mac" data-venv="">python -m venv sklearn-venv</span
+        ><span class="sk-expandable" data-packager="pip" data-os="linux" data-venv="">source sklearn-venv/bin/activate</span
+        ><span class="sk-expandable" data-packager="pip" data-os="mac" data-venv="">source sklearn-venv/bin/activate</span
+        ><span class="sk-expandable" data-packager="pip" data-os="windows" data-venv="">sklearn-venv\Scripts\activate</span
+        ><span class="sk-expandable" data-packager="pip" data-venv="">pip install -U scikit-learn</span
+        ><span class="sk-expandable" data-packager="pip" data-os="mac" data-venv="no">pip install -U scikit-learn</span
+        ><span class="sk-expandable" data-packager="pip" data-os="windows" data-venv="no">pip install -U scikit-learn</span
+        ><span class="sk-expandable" data-packager="pip" data-os="linux" data-venv="no">pip3 install -U scikit-learn</span
+        ><span class="sk-expandable" data-packager="conda" data-venv="">conda create -n sklearn-env</span
+        ><span class="sk-expandable" data-packager="conda" data-venv="">conda activate sklearn-env</span
+        ><span class="sk-expandable" data-packager="conda">conda install scikit-learn </span
+       ></code></pre></div>
+
+In order to check your installation you can use
+
+.. raw:: html
+
+   <div class="highlight"><pre><code
+      ><span class="sk-expandable" data-packager="pip" data-os="linux" data-venv="no">python3 -m pip show scikit-learn  # to see which version and where scikit-learn is installed</span
+      ><span class="sk-expandable" data-packager="pip" data-os="linux" data-venv="no">python3 -m pip freeze  # to see all packages installed in the active virtualenv</span
+      ><span class="sk-expandable" data-packager="pip" data-os="linux" data-venv="no">python3 -c "import sklearn; sklearn.show_versions()"</span
+      ><span class="sk-expandable" data-packager="pip" data-venv="">python -m pip show scikit-learn  # to see which version and where scikit-learn is installed</span
+      ><span class="sk-expandable" data-packager="pip" data-venv="">python -m pip freeze  # to see all packages installed in the active virtualenv</span
+      ><span class="sk-expandable" data-packager="pip" data-venv="">python -c "import sklearn; sklearn.show_versions()"</span
```

Comment:
This `python -c "import sklearn; sklearn.show_versions()"` should also be visible when conda is the package manager (with or without conda env).

Reply:
> For buttons, the cursor should change on hover. (By default it is a hand with a glove on it).

Fixed in f0be47c995696dae075965ad1b7240b2793b3e7a

Reply:
Done in f0be47c995696dae075965ad1b7240b2793b3e7a, but not sure if on linux with conda python 3 is `python` or `python3`.

Reply:
When using conda on Linux, it actually depends on whether or not you decide to activate the `base` environment in `.bashrc`. One can assume yes to keep things simple.